cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/SlangTextualKeys.java

```diff
@@ -29,10 +29,13 @@
     String ON_FAILURE_KEY = "on_failure";
 
     //action
-    String JAVA_ACTION = "java_action";
-
-    //operation
-    String ACTION_KEY = "action";
+    String JAVA_ACTION_KEY = "java_action";
+    String JAVA_ACTION_CLASS_NAME_KEY = "class_name";
+    String JAVA_ACTION_METHOD_NAME_KEY = "method_name";
+    String JAVA_ACTION_GAV_KEY = "gav";
```

Comment:
I do not think that it should be called gav... It's a Maven term, not a Java term
I think we should call it something like `artifact:` and maybe support a value in the syntax of gav, or the simple artifact name in case the CloudSlang user does not use Maven